packages/ketcher-core/src/domain/entities/sgroup.ts

```diff
@@ -296,6 +296,8 @@ export class SGroup {
   }
 
   static removeAtom(sgroup: SGroup, aid: number): void {
+    if (!sgroup) return
```

Comment:
Hmmm, not sure if it is a good place to do that. I mean a function is quite simple and I think, if it is not provided first argument, then higher function in stack didn't provide it – and that is the real problem.